proposals/run_eoepca_externally.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "run_eoepca_externally",
+    "summary": "Run processing externally on EOEPCA",
+    "description": "Runs processing tasks on an [EOEPCA](https://eoepca.org/) compliant processing platform, which essentially runs a hosted docker container.",
```

Comment:
We should probably add information about the API interface. Is this always OGC API - Processes?

Reply:
It needs to be much more specific than just saying OGC API - Processes.
EOEPCA should have a kind of profile with process signatures that an application package can support to be callable by openEO.